frontend/playwright.config.ts

```diff
@@ -23,7 +23,7 @@ export default defineConfig({
   /* Retry on CI only */
   retries: process.env.CI ? 2 : 0,
   /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : undefined,
```

Comment:
[blocker/question]: why  increasing the workers now ?

Reply:
This makes the CI tests for Playwright faster, since the machine we use for GH Actions can easily accommodate at least 4 workers to run tests in parallel. All of our e2e tests are isolated, so we can run them in parallel.